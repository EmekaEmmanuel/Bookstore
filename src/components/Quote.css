* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  font-family: 'Open Sans', sans-serif;
}

.quote_article {
  padding: 150px 0;
  width: 90%;
  margin: auto;
  display: flex;
  justify-content: flex-start;
  font-size: 35px;
  align-items: center;
}
