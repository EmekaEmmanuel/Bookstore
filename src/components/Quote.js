import React from 'react';
import Nav from './Nav';
import './Quote.css';

function Quote() {
  /* eslint-disable max-len */
  return (
    <div>
      <Nav />
      <article className="quote_article">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!.  Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!. Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!.
      </article>
    </div>
  );
  /* eslint-enable max-len */
}

export default Quote;
