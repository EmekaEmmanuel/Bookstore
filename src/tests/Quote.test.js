import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Quote />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
