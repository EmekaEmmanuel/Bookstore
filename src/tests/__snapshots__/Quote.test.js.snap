// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="nav_container"
  >
    <header
      className="nav_header"
    >
      <h1>
        Math Magicians
      </h1>
      <nav
        className="nav_list"
      >
        <li>
          <a
            className="nav_anchor td_none"
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
        </li>
        <li
          className="border_active"
        >
          <a
            className="nav_anchor  td_none"
            href="/Calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            className="nav_anchor td_none"
            href="/Quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </nav>
    </header>
  </div>
  <article
    className="quote_article"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!.  Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!. Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit nesciunt consequatur eos similique quia!.
  </article>
</div>
`;
